VERSION.FUSE.txt

```diff
@@ -0,0 +1 @@
+3.0.0
```

Comment:
This won't be needed in Nethermind codebase?

Reply:
This is just use by the CICD pipeline for version control of docker container so yeah won't be in PR back into NM 😊